### ISSUE TITLE
We need a test to make sure that Viper calls stay off critical paths

### ISSUE THREAD

**Issue by bcbrock** [hyperledger-archives/fabric#1665]
## Description

Now that we have solved #647, we need a way to make sure that Viper calls for cached variables don't creep back into critical paths, .e.g. a Go "unit test" or a script executed during CI.

**Comment by corecode** [comment]
I believe that viper is the wrong library for configuration.  See #1309.

**Comment by bcbrock** [comment]
@corecode I agree it has many flaws, but until we change it it won't be that much work to make sure we are using the caching that was implemented.
